coderd/rbac/scopes_catalog_internal_test.go

```diff
@@ -27,6 +28,17 @@ func TestExternalScopeNames(t *testing.T) {
 			continue
 		}
 
+		// Composite coder:* scopes expand to one or more site permissions.
+		if strings.HasPrefix(name, "coder:") {
+			s, err := ScopeName(name).Expand()
+			require.NoErrorf(t, err, "catalog entry should expand: %s", name)
+			require.NotEmpty(t, s.Site)
```

Comment:
We should assert the expected permissions here based on the scope name.

Reply:
done